LR_Tourist/BLL/Startup.cs

```diff
@@ -0,0 +1,30 @@
+﻿using AutoMapper;
+using BLL;
+using DA;
+using DA.Services.Repository;
+using Microsoft.Extensions.DependencyInjection;
+using System.Reflection;
+
+namespace DI
+{
+    public static class Startup
+    {
+        public static ServiceProvider Configure()
```

Comment:
It looks very suspicious.
Do you use NInject or Microsoft DI?
**Cleanup useless packages and refactor registrations.**

PS: If you use Microsoft DI - it would be better to have all registrations in one place (in `Tourist` project).

Reply:
I think this file should be a part of `Tourist` project anyway.

Also, investigate and fix your project dependencies.

Reply:
maybe BLL)))

Reply:
No.